docs/learn/streams.md

```diff
@@ -16,15 +16,51 @@
 
```

Comment:
nit: above reference doesn't have all providers, which means the providers also likely don't have backlinks, please add them